includes/wikia/api/NavigationApiController.class.php

```diff
@@ -30,11 +30,6 @@ public function getData(){
 			[ 'urlFields' => 'href' ],
 			NavigationModel::CACHE_TTL
 		);
-		$errors = $model->getErrors();
-
-		if ( !empty( $errors ) ) {
```

Comment:
This method is only used by MercuryApi.
So it's safe to remove this part.

Exception is no longer thrown to unify behavior across Mercury and Oasis.
In Oasis when model has errors data attribute is added to navigation HTML node and this attr is only read in edit mode.

Without this change Mercury was displaying empty nav in case of errors inside model.
With this change Mercury should display the same thing as Oasis